x-pack/test/stack_functional_integration/apps/maps/_maps.ts

```diff
@@ -14,11 +14,13 @@ export default function ({
 }: FtrProviderContext & { updateBaselines: boolean }) {
   const screenshot = getService('screenshots');
   const browser = getService('browser');
+  const find = getService('find');
   const PageObjects = getPageObjects(['maps']);
 
   describe('check Elastic Maps Server', function () {
     before(async function () {
       await PageObjects.maps.loadSavedMap('EMS Test');
+      await find.clickByButtonText('Dismiss');
```

Comment:
I used this since there isn't any data-test-subj for that button. once we add a test to check it works we can add a data-test-subj for it.